pkg/lib/page.scss

```diff
@@ -76,29 +76,46 @@ a.disabled:hover {
 }
 
 /* Animation of new items */
-
 .new-item {
     animation: newRow 4s ease-in;
 }
 
+:root {
+    --ct-animation-new-background: #fdf4dd;
+}
+
+/* Simple fade animation by default (mainly for reduced motion) */
 @keyframes newRow {
   0% {
-    background: #fdf4dd;
-    transform: scaleX(0) scaleY(0);
-    opacity: 0;
+    background-color: var(--ct-animation-new-background);
   }
-  10% {
-    transform: scaleX(1) scaleY(1);
+  50% {
+    background-color: var(--ct-animation-new-background);
   }
-  12% {
-    transform: scaleX(1) scaleY(1.25);
-    opacity: 1;
-  }
-  15% {
-    transform: scaleX(1) scaleY(1);
-  }
-  70% {
-    background: #fdf4dd;
+}
```

Comment:
So it starts with the background color, then doesn't do anything for half the time then fades out over 2 seconds.

The one below does a bunch of other stuff, but uses the same technique.

I adjusted the timing because it doesn't need to accommodate the moving animation time and then it also makes the animation slightly more gradual.

(Prefers reduced motion is not meant to completely remove all animations. It's meant to have little to no movement and not have drastic blinking. Mild, slow movements are even OK, although I went with just a simple fade for reduced motion.)

All major OSes have means to remove motion, usually under accessibility. I tested this on Firefox, which acknowledges GNOME's animation setting (in accessibility).

Reply:
Just explaining how this all works, especially on the default reduced motion, with more animation on the allows animation ('no preference") part.